app/controllers/api/internal/v1/competitions_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Api::Internal::V1::CompetitionsController < Api::Internal::V1::ApiController
+  # We are using our own authentication method with vault
+  protect_from_forgery except: [:show, :qualifications]
+
+  def show
+    competition = competition_from_params
+
+    if stale?(competition)
+      options = {
+        only: %w[id name website start_date registration_open registration_close announced_at cancelled_at end_date competitor_limit
+                 extra_registration_requirements enable_donations refund_policy_limit_date event_change_deadline_date waiting_list_deadline_date
+                 on_the_spot_registration on_the_spot_entry_fee_lowest_denomination qualification_results event_restrictions
+                 base_entry_fee_lowest_denomination currency_code allow_registration_edits allow_registration_self_delete_after_acceptance
+                 allow_registration_without_qualification refund_policy_percent use_wca_registration guests_per_registration_limit venue contact
+                 force_comment_in_registration use_wca_registration external_registration_page guests_entry_fee_lowest_denomination guest_entry_status
+                 information events_per_registration_limit],
+        methods: %w[url website short_name city venue_address venue_details latitude_degrees longitude_degrees country_iso2 event_ids registration_currently_open?
+                    main_event_id number_of_bookmarks using_payment_integrations? uses_qualification? uses_cutoff? competition_series_ids],
+        include: %w[delegates organizers tabs],
+      }
+      render json: competition.as_json(options)
```

Comment:
Do we not have to_competition_info for this?

Reply:
Ah cool! Wasn't aware of this as the v0 competition API doesn't make use of it either - but have updated both to use `to_competition_info`

Reply:
You should check if it has all the fields that we want because we might have added one since I created it

Reply:
I did a detailed check, and `to_competition_info` had more information than the old method - so we're definitely not losing any fields. I didn't save the comparison so can't remember exactly what the field was, but I didn't see any issue with exposing it on the non-authenticated route as well.